tests/suites/casper/formfill.js

```diff
@@ -219,7 +236,7 @@ casper.test.begin('getFormValues() tests', 2, function(test) {
     var fpath = fs.pathJoin(phantom.casperPath, 'README.md');
     var fileValue = 'README.md';
     if (phantom.casperEngine === 'phantomjs') {
-        if (utils.matchEngine({
+        if (!utils.matchEngine({
             name: 'phantomjs',
```

Comment:
Reverted this change. I have left max in there. The getFormValues() fakepath bug seems to  affect only 2.0.0. So now we skip the test for "file multiple" in 2.0.0 similar to other file tests.